ChangeLog.md

```diff
@@ -1,3 +1,9 @@
+# Changes in 1.0.3
+
+* Fix an inconsistence in `clap_plugin->destroy()` documentation:
+  it is now **required** to deactivate the plugin before destroying it.
```

Comment:
```suggestion
* Fix an inconsistency in `clap_plugin->destroy()` documentation:  
  It is now **required** to deactivate the plugin before destroying it.
```

`inconsistence` -> `inconsistency`, and Markdown requires two spaces at the end of a line for a `<br>` line break.

Reply:
It was one sentence, I did not expect to break line.
Yes for the spelling mistake.